global.json

```diff
@@ -1,9 +1,14 @@
 {
   "tools": {
-    "dotnet": "3.1.101"
+    "dotnet": "5.0.100-preview.5.20251.2",
+    "runtimes": {
+      "dotnet/x64": [
```

Comment:
Why only for `x64`? We should need it for all archs, right?

Reply:
I was actually just testing the syntax :) and didn't know if this would be sufficient to make CI pass. Turns out it did and it auto-merged.

Reply:
Right, but I think we should do it for all arch’s just in case someone needs to build/run tests in another arch.

Reply:
Yup yup, I agree. I just opened https://github.com/dotnet/corefxlab/pull/2912 and tagged you